docs/concepts/feature-definition.md

```diff
@@ -65,24 +60,23 @@ request_anchor = FeatureAnchor(name="request_features",
                                features=features)
 ```
 
-
 For the features field above, there are two different types, simple anchored features and window aggregation features.
 
 ### Simple anchored features
 
-1) For simple anchored features, see the example below:
+1. For simple anchored features, see the example below:
 
 ```python
 f_trip_time_duration = Feature(name="f_trip_time_duration",
                                feature_type=INT32,
                                transform="time_duration(lpep_pickup_datetime, lpep_dropoff_datetime, 'minutes')")
 ```
 
-Note that for `transform` section, you can put a simple expression to transform your features. For more information, please refer to [Feathr Expression Language](../how-to-guides/expression-language.md).
+Note that for `transform` section, you can put a simple expression to transform your features. For more information, please refer to [Feathr User Defined Functions (UDFs)](../how-to-guides/feathr-udfs.md).
```

Comment:
Same here, shall we update to `https://linkedin.github.io/feathr/` links

Reply:
It should automatically be compiled to the `https://linkedin.github.io/feathr/` links once it's published so it should be fine